internals/generators/organization/config.yml.hbs

```diff
@@ -323,6 +324,20 @@ client:
         text:
           en: "Go back to homepage"
 
+    captive_portal_form:
+      method: post
+      action: http://localhost/
+      fields:
+        username: auth_user
+        password: auth_pass
+      additional_fields:
+        - name: zone
+          value: zone_name
+        - name: redirurl
+          value: http://0.0.0.0:8080/{{slug}}/status
```

Comment:
isn't `localhost` or `127.0.0.1` more common? It may be less confusing for most users.

Reply:
Currently it's passed as an array field and it's name can be different for different captive portals, so I don't have any way to identify this field and modify it's value on fronted app. We can ask for captive portal's root url and save it in yml file and then use that variable in all the places where relative url is required.